tests/nft/nft.cc

```diff
@@ -3487,19 +3483,6 @@ TEST_CASE("mata::nft::get_useful_states_tarjan") {
 		CHECK(bv == ref);
 	}
 
-    SECTION("from regex (a+b*a*)") {
-        Nft aut;
-        mata::parser::create_nfa(&aut, "(a+b*a*)", false, EPSILON, false);
-
-        mata::BoolVector bv = aut.get_useful_states();
-        mata::BoolVector ref({ 1, 0, 1, 0, 1, 0, 1, 0, 0});
-        CHECK(bv == ref);
-
-        aut = reduce(aut.trim());
-        bv = aut.get_useful_states();
-        CHECK(bv == mata::BoolVector({ 1, 1, 1, 1}));
-    }
-
```

Comment:
Why was this test removed entirely?

Reply:
It must use `create_nfa` as it does not want to reduce the NFA created from the regex, so I couldn't replace with `create_from_regex`. However, it also expects a specific structure of the created NFA from a regex, which is weird, it assumes that it works specific to RE2 parser.

Reply:
Yeah, the checked conditions are silly and check implementation details instead of any reasonable qualities of the regex. OK, let us remove the test altogether. I do not think there is anything special not already checked by the other tests in this particular regex.